tests/trainer/test_trainer.py

```diff
@@ -1150,7 +1150,7 @@ def test_can_resume_training(self):
         # won't be the same since the training dataloader is shuffled).
 
         with tempfile.TemporaryDirectory() as tmpdir:
-            kwargs = dict(output_dir=tmpdir, train_len=128, save_steps=5, learning_rate=0.1)
+            kwargs = dict(output_dir=tmpdir, train_len=128, save_steps=5, learning_rate=0.1, logging_steps=5)
```

Comment:
This tests the fix as the epoch is logged and the logs are recorded in the trainer state and equality between states is tested.